flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -2721,6 +2721,14 @@ void OmpAttributeVisitor::CheckObjectInNamelist(
         "Variable '%s' in NAMELIST cannot be in a %s clause"_err_en_US,
         name.ToString(), clauseName.str());
   }
+
+  if (stmtFunctionExprSymbols_.find(ultimateSymbol) !=
+      stmtFunctionExprSymbols_.end()) {
+    context_.Say(name.source,
+        "Variable '%s' in STATEMENT FUNCTION expression cannot be in a "
```

Comment:
"statement function" isn't literal Fortran syntax, don't capitali[sz]e things that are not keywords.

And please don't break error message strings across multiple lines -- that makes it much more difficult to find them with grep.

Reply:
Thanks for taking a look. I have removed the capitalization of "statement function" here.

I left the error message string across multiple lines because the next word in the string is `%s` anyway.